lib/elixir/src/elixir_parser.yrl

```diff
@@ -552,7 +552,8 @@ Erlang code.
 -import(lists, [reverse/1, reverse/2]).
 
 meta(Line, Counter) -> [{counter,Counter}|meta(Line)].
-meta(Line) when is_integer(Line) -> [{line,Line}];
+meta([Line, Column, EndColumn]) when is_integer(Line), is_integer(Column), is_integer(EndColumn) -> [{line, Line}];
+meta(Line) when is_integer(Line) -> [{line, Line}];
```

Comment:
Is there a chance we are passing only an integer still? If so, wouldn't it be a bug? :)